packages/web-components/src/radio-group/radio-group.stories.ts

```diff
@@ -0,0 +1,162 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../helpers.stories.js';
+import { RadioGroup as FluentRadioGroup } from './radio-group.js';
+import './define.js';
+import '../radio/define.js';
+import { RadioGroupOrientation } from '@microsoft/fast-foundation';
+
+type RadioGroupStoryArgs = Args & FluentRadioGroup;
+type RadioGroupStoryMeta = Meta<RadioGroupStoryArgs>;
+
+const storyTemplate = html<RadioGroupStoryArgs>`
+  <fluent-radio-group
+    aria-labelledby="label-1"
+    ?disabled=${(x: { disabled: boolean }) => x.disabled}
+    ?stacked=${(x: { stacked: boolean }) => x.stacked}
+    orientation=${(x: { orientation: 'vertical' | 'horizontal' }) => x.orientation}
```

Comment:
Let's keep this simple as is elsewhere (all the way down) where we simply reference "x". I don't think we need to enumerate the types (which don't use the actual types in this case...) in the story file.